### ISSUE TITLE
Add lines to chart

### ISSUE THREAD

**Issue by OstapchelaEst** [amcharts/amcharts5#1604]
Hi. I am trying to implement a functionality which is to add a line between two time points. For this I tried to use the [trendline approach](https://www.amcharts.com/demos/trend-lines/). But in my case the end point of the line can go beyond the main line and in this case the time range changes and there is an offset. I need a behavior where it will not change and will remain the same as before the line was added. You can see this behavior [here](https://jsfiddle.net/ymerwk0g/21/) by clicking on the "add line" button. 

Also in this example I tried to add a line using am5.Line.new but that didn't work either.

Maybe you can give me some advice?

**Comment by martynasma** [comment]
You can use series' setting `ignoreMinMax: true` to make series not influence the actual scale:

```JavaScript
  var series = chart.series.push(
    am5xy.LineSeries.new(root, {
      xAxis: xAxis,
      yAxis: yAxis,
      valueXField: "date",
      stroke: color,
      valueYField: "value",
      ignoreMinMax: true
    })
  );
```

**Comment by OstapchelaEst** [comment]
@martynasma 
Thank you so much it really did work. I modified the code a bit and added "bullets" to the lines that are added. If the lines are close to each other, the bullets overlap each other. I tried to use the `stacked` property but it doesn't seem to work for "bullets" in different series. Is there any way to shift them relative to each other when they overlap?
https://jsfiddle.net/bnkz1ao5/34/
